frontend/src/components/PlannerCart/CartCourseCard.jsx

```diff
@@ -31,34 +33,36 @@ const CartCourseCard = ({ code, title, showAlert }) => {
   };
 
   return (
-    <div className="planner-cart-card">
-      <div role="menuitem" onClick={handleCourseLink} style={{ cursor: "pointer" }}>
-        <Text className="text" strong>
-          {code}:{" "}
-        </Text>
-        <Text className="text">{title}</Text>
-      </div>
-      <div className="planner-cart-card-actions">
-        <Popconfirm
-          placement="bottomRight"
-          title="Remove this course from your planner?"
-          onConfirm={confirmDelete}
-          style={{ width: "200px" }}
-          okText="Yes"
-          cancelText="No"
-        >
-          <Tooltip title={`Remove ${code}`}>
-            <Button
-              danger
-              size="small"
-              shape="circle"
-              icon={<DeleteOutlined />}
-              loading={loading}
-            />
-          </Tooltip>
-        </Popconfirm>
-      </div>
-    </div>
+    <ThemeProvider theme={theme === "light" ? lightTheme : darkTheme}>
```

Comment:
You don't need to define `ThemeProvider` here? It should be already used in the top level component. Remove `ThemeProvider`